src/composition.jl

```diff
@@ -42,6 +41,16 @@ for (f, _f, g) in ((:issymmetric, :_issymmetric, :transpose),
     end
 end
 
+# A * B * A and A * B * A' are positive definite if (sufficient condition) A & B are positive definite
+LinearAlgebra.isposdef(A::CompositeMap) = _isposdef(A.maps)
+_isposdef(maps::Tuple{}) = true # empty product is equivalent to "I" which is pos. def.
+_isposdef(maps::Tuple{<:LinearMap}) = isposdef(maps[1])
+function _isposdef(maps::Tuple{Vararg{LinearMap}})
+    (maps[end] == adjoint(maps[1]) || maps[end] == maps[1]) && 
+    isposdef(maps[1]) && isposdef(maps[end]) &&
+    _isposdef(Base.front(Base.tail(maps)))
```

Comment:
I believe it suffices to check one of the foregoing conditions are met?
```suggestion
    isposdef(maps[1]) && _isposdef(Base.front(Base.tail(maps)))
```